### ISSUE TITLE
Homekit node-red nodes not working in Hass.io

### ISSUE THREAD

**Issue by Tim3100** [home-assistant/hassio#763]
Hi,

installing Homekit related nodes in node-red gives an error on installation in Hass.io (see below). As this is happening for several modules, I guess it might be an issue of Hass.io.

node-red-contrib-homekit-bridged install error:
> curve25519-n@1.4.0 install /config/node-red/node_modules/curve25519-n↵> node-gyp rebuild↵↵gyp ERR! build error ↵gyp ERR! stack Error: not found: make↵gyp ERR! stack at getNotFoundError (/usr/lib/node_modules/npm/node_modules/which/which.js:13:12)↵gyp ERR! stack at F (/usr/lib/node_modules/npm/node_modules/which/which.js:68:19)↵gyp ERR! stack at E (/usr/lib/node_modules/npm/node_modules/which/which.js:80:29)↵gyp ERR! stack at /usr/lib/node_modules/npm/node_modules/which/which.js:89:16↵gyp ERR! stack at /usr/lib/node_modules/npm/node_modules/which/node_modules/isexe/index.js:42:5↵gyp ERR! stack at /usr/lib/node_modules/npm/node_modules/which/node_modules/isexe/mode.js:8:5↵gyp ERR! stack at FSReqWrap.oncomplete (fs.js:152:21)↵gyp ERR! System Linux 4.14.66-v8↵gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"↵gyp ERR! cwd /config/node-red/node_modules/curve25519-n↵gyp ERR! node -v v8.11.4↵gyp ERR! node..."

node-red-contrib-apple-tv:
npm WARN notice [SECURITY] lodash has the following vulnerability: 1 low. Go here for more details: https://nodesecurity.io/advisories?search=lodash&version=3.10.1 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.

> sodium@2.0.3 preinstall /config/node-red/node_modules/sodium
> node install.js --preinstall

/bin/sh: make: not found
/config/node-red/node_modules/sodium/install.js:287
            throw new Error(cmdLine + ' exited with code ' + code);
            ^

Error: make libsodium exited with code 127
    at ChildProcess.<anonymous> (/config/node-red/node_modules/sodium/install.js:287:19)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sodium@2.0.3 preinstall: `node install.js --preinstall`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the sodium@2.0.3 preinstall script.
np...

**Comment by frenck** [comment]
This is not an Hass.io issue and therefore this issue is created on the wrong GitHub repository.

About the error: `/bin/sh: make: not found`

You need to add make to the system packages of the add-on configuration option.

For support on third-party add-ons, please check with the add-on authors themselves.

**Comment by Tim3100** [comment]
Sorry for putting this at the wrong location and thanks for the hint. I followed your advice and added quite some dependencies to fix follow-up errors. Unfortunately, I got stock with an error stating 
make: Entering directory '/opt/node_modules/mdns/build'
  CXX(target) Release/obj.target/dns_sd_bindings/src/dns_sd.o
In file included from ../src/dns_sd.cpp:1:0:
../src/mdns.hpp:32:20: fatal error: dns_sd.h: No such file or directory 
[I think during building libsodium]
I tried adding libavahi-compat-libdnssd-dev, but this doesn't work. I will post it on respective GitHub repo.
